server/src/socket/controller/GameController.ts

```diff
@@ -161,5 +161,40 @@ export class GameController extends SockerController {
   cheater() {
     this.io.emit('cheater');
   }
+
+  async playerConnected(token, gameId: number) {
+    const gameRoom = `game/${gameId}`;
+    let user;
+    try {
+      user = verifyToken(token);
+    } catch (error) {
+      return;
+    }
+
+    const game = await this.gameRepository.findOne(gameId);
+    const gameState = await this.gameStateRepository.findOne({
+      where: { game },
+    });
+    if (!game || !gameState) {
+      // WTF
+      console.error(
+        `user ${user.userId} attempted to connect to game ${game.id}, but it doesn't exist`
+      );
+      return;
+    }
+    this.socket.join(gameRoom);
+    const isPlayer1 = game.player1.id === user.userId;
+    const isPlayer2 = game.player2.id === user.userId;
+    if (isPlayer1 || isPlayer2) {
```

Comment:
if one of our players in the game connects, we increment the counter. if both have connected, START THE GAME. 
other people can still connect to the game, but they should see the in progress game, not the intro